src/OrleansAWSUtils/Storage/Provider/DynamoDBStorageProvider.cs

```diff
@@ -0,0 +1,340 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Orleans.Providers;
+using Orleans.Runtime;
+using Newtonsoft.Json;
+using System.Threading;
+using Orleans.Serialization;
+using Amazon.DynamoDBv2.Model;
+using Amazon.DynamoDBv2;
+using OrleansAWSUtils;
+using System.IO;
+using OrleansAWSUtils.Storage;
+
+namespace Orleans.Storage
+{
+    public class DynamoDBStorageProvider : IStorageProvider
```

Comment:
All provider implementations are never accessed directly so no-one will ever read comments. The only reason it is public is so it can be accessed by Orleans internals. I think is more useful a better documentation on the docs page.